### ISSUE TITLE
Feature Request: Dectar entre caracteres especiales y no especiales en la busqueda

### ISSUE THREAD

**Issue by Eduardoveras** [developersdo/opensource#65]
Por ejemplo, que las personas con el nombre josé en github salgan cuando se busca jose (sin tilde)

**Comment by DannyFeliz** [comment]
Pero Github no te va a dejar utilizar un nombre de usuario con caracteres especiales, solo alfanuméricos y guión tal como se muestra en la imagen
![asd](https://image.ibb.co/dPjrt6/Screen_Shot_2017_10_26_at_10_01_19_AM.png)

**Comment by rmariuzzo** [comment]
Cierto, pero la búsqueda se hace ahora mismo por el nombre y no por el
login.

**Comment by rmariuzzo** [comment]
Ah, entiendo.

**Comment by rmariuzzo** [comment]
@DannyFeliz el filtro utiliza el nombre y no el login, pienso que hay que utilizar ambas cosas. Pero sobre todo, como mencionó @Eduardoveras tenemos que buscar hacer búsquedas sin y con acento.

**Comment by rmariuzzo** [comment]
Acabo de ver que a este feature se le llama “accent folding”, más detalles: http://alistapart.com/article/accent-folding-for-auto-complete

**Comment by rmariuzzo** [comment]
I'm working on this.

**Comment by Eduardoveras** [comment]
Thanks!
